stages/org.osbuild.bootc.install-to-filesystem

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+"""Run bootc install to-filesystem
+
+Note that this needs the disk.img in the inputs as one continous
+devices so that bootupd can install grub to the mbr.
+
+Buildhost commands used: bootc
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util import containers
+
+CAPABILITIES = ["CAP_MAC_ADMIN"]
+
+SCHEMA_2 = r"""
+"inputs": {
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["images"],
+  "properties": {
+    "images": {
+      "type": "object",
+      "additionalProperties": true
+    }
+  }
+},
+"options": {
+  "additionalProperties": false
+},
+"devices": {
+  "type": "object",
+  "additionalProperties": true
+},
+"mounts": {
+  "type": "array"
+}
+"""
+
+
+def main(inputs, mounts):
+    images = containers.parse_containers_input(inputs)
+    assert len(images) == 1
+    image = list(images.values())[0]
+
+    with containers.container_source(image) as (_, source):
+        env = os.environ.copy()
+        # without this fallback this will fail on any non-selinux system
+        env["BOOTC_SETENFORCE0_FALLBACK"] = "1"
```

Comment:
Hmm...can you be more precise with "non-selinux system"?   When you're testing osbuild on Debian w/o SELinux or something?  If so I'm a bit confused how the `setenforce` fallback would work, but the `install_t` thing wouldn't.

Reply:
Maybe I should have been more clear in the comment, sorry for this! So this is just a draft right now as there are some problems with this PR around testing. We currently test (among others) on GH runners which do not have selinux enabled so I wanted to see if we strictly need selinux enabled hosts to run this stage on.

Reply:
Right, same thing e.g. here: https://github.com/containers/bootc/blob/ba8f744a066094aebda1a2b0a27dd93014f18ffa/.github/workflows/ci.yml#L163
But there we explicitly pass `--disable-selinux` *only* in a CI flow.  I think I'd advocate something similar, where e.g. we set an environment variable like `OSBUILD_CI_DISABLE_SELINUX` in the GHA workflow that in turn adds `--disable-selinux` here.

That said I am now curious if this `setenforce 0` actually really works.

Reply:
I actually hit this GHA issue a while ago and wrote up https://github.com/ostreedev/ostree/issues/2804 which has some more info on the ostree (and similar bootc) side that links to the usage of `setfscreatecon` versus simply `fsetxattr(security.selinux)`.